app/config.go

```diff
@@ -184,30 +186,36 @@ func InitConfig(datadir, tmNode, persistentPeers, seeds, tmRPCPort, tmPeersPort
 	if _, err := os.Stat(configFilepath); err == nil {
 		jsonFile, err = os.OpenFile(configFilepath, os.O_RDWR, os.ModePerm)
 		if err != nil {
-			panic("cannot open config json file: " + err.Error())
+			fmt.Println("cannot open config json file: " + err.Error())
+			os.Exit(1)
```

Comment:
```suggestion
			log.Fatalf("cannot open config json file: %s",  err.Error())
```

stdlib package `log` offers `log.Fatalf` which logs the error with a timestamp & calls onto os.Exit(1)

the `log` package needs to also be added on this file for this suggestion to work